src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -0,0 +1,175 @@
+        additionalErrorMessage: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    walletAdditionalDetails: {
+        errorFields: [],
+        loading: false,
+        additionalErrorMessage: '',
+    },
+};
+
+class AdditionalDetailsStep extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.requiredText = props.translate('additionalDetailsStep.isRequiredField');
+        this.fields = [
+            {
+                label: props.translate('additionalDetailsStep.legalFirstNameLabel'),
+                fieldName: 'legalFirstName',
+            },
+            {
+                label: props.translate('additionalDetailsStep.legalMiddleNameLabel'),
+                fieldName: 'legalMiddleName',
+            },
+            {
+                label: props.translate('additionalDetailsStep.legalLastNameLabel'),
+                fieldName: 'legalLastName',
+            },
+            {
+                label: props.translate('additionalDetailsStep.addressLabel'),
+                fieldName: 'addressStreet',
+            },
+            {
+                label: props.translate('additionalDetailsStep.cityLabel'),
+                fieldName: 'addressCity',
+            },
+            {
+                label: props.translate('additionalDetailsStep.stateLabel'),
+                fieldName: 'addressState',
+            },
+            {
+                label: props.translate('additionalDetailsStep.zipCodeLabel'),
+                fieldName: 'addressZip',
+            },
+            {
+                label: props.translate('additionalDetailsStep.phoneNumberLabel'),
+                fieldName: 'phoneNumber',
+            },
+            {
+                label: props.translate('additionalDetailsStep.dobLabel'),
+                fieldName: 'dob',
+            },
+            {
+                label: props.translate('additionalDetailsStep.ssnLabel'),
+                fieldName: 'ssn',
+            },
+        ];
+
+        this.state = {
+            firstName: '',
+            legalMiddleName: '',
+            legalLastName: '',
+            address: '',
+            city: '',
+            state: '',
+            zipCode: '',
+            phoneNumber: '',
+            dob: '',
+            ssn: '',
+        };
+    }
+
+    render() {
+        const errorFields = this.props.walletAdditionalDetails.errorFields || [];
+        return (
+            <ScreenWrapper>
+                <KeyboardAvoidingView style={[styles.flex1]} behavior="height">
+                    <HeaderWithCloseButton
+                        title={this.props.translate('additionalDetailsStep.headerTitle')}
+                        onCloseButtonPress={() => Navigation.dismissModal()}
+                    />
+                    <View style={[styles.flex1]}>
+                        <View style={[styles.ph5]}>
+                            <Text style={styles.mb3}>{this.props.translate('additionalDetailsStep.helpText')}</Text>
```

Comment:
NAB but pet-peevy that we're not consistent about whether to include square brackets when there's only a single style prop. My vote would be to just always include square brackets. Can discuss later and create a style guide regulation for this.

Reply:
Yeah, there is a big readme about styling: https://github.com/Expensify/Expensify.cash/blob/main/STYLING.md but it doesn't mention anything about consistency in regards to this.